connector.py

```diff
@@ -0,0 +1,53 @@
+"""
+Copyright 2019 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+
+def get_ephemeral(service, project, instance, pub_key):
+    """
+    A helper function that requests an ephemeral certificate from the
+    Cloud SQL Instance.
+
+    Takes in a service object, the project name, the instance name and
```

Comment:
Would you mind putting this into the google format for docstrings? :) That will add info on the types of the arguments and should make it easier for other folks to use. You'll also be able to get nice generated docs.

Example: https://github.com/googleapis/google-cloud-python/blob/3d324389b92d43e52486f0fe2aca8b41e950640c/bigquery_storage/google/cloud/bigquery_storage_v1beta1/client.py#L85-L115
Documentation on google style docstrings: http://google.github.io/styleguide/pyguide.html#381-docstrings

Reply:
Will do, thanks for the links!